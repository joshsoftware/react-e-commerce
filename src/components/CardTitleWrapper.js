import React from 'react';
import { CardTitle } from 'reactstrap';
import { PropTypes } from 'prop-types';

const CardTitleWrapper = ({ title, className }) => {
  return <CardTitle className={className}>{title}</CardTitle>;
};

export default CardTitleWrapper;

CardTitleWrapper.propTypes = {
  title: PropTypes.string.isRequired,
  className: PropTypes.string
};
