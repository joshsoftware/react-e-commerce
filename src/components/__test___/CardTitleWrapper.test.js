import React from 'react';
import { render } from '@testing-library/react';
import CardTitleWrapper from '../CardTitleWrapper';

describe('Card Title', () => {
  it('Must Return Card Title', () => {
    const title = 'Title';
    const { asFragment } = render(<CardTitleWrapper title={title} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
