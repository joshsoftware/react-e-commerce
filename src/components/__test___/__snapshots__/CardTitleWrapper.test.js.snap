// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Title Must Return Card Title 1`] = `
<DocumentFragment>
  <div
    class="card-title"
  >
    Title
  </div>
</DocumentFragment>
`;
